tests/__init__.py

```diff
@@ -0,0 +1,24 @@
+def elasticsearch_running():
```

Comment:
On a similar note, when ES is not running some of the tests fail with connection errors